terraso_backend/apps/project_management/models/sites.py

```diff
@@ -85,17 +74,17 @@ class Meta(BaseModel.Meta):
     )
 
     @property
-    def owned_by_user(self):
+    def is_unaffiliated(self):
```

Comment:
I refactored this method name to match the language used by the requirements docs.